switches/http2/http/HttpDispatcher.h

```diff
@@ -98,6 +99,9 @@ namespace Aseba
 		typedef std::map<HttpMethod, URIHandlerMap> HandlersMap;
 		//! Handlers for known method + URI couples
 		HandlersMap handlers;
+		//! Map of method, split URI to JSON-formatted OAS documentation
+		typedef std::map<HttpMethod, std::map<strings, json>> ApidocsMap;
```

Comment:
There is one entry of this map for one entry in `handlers`, right? If so I would build a `pair<Handler, json>` as key in handlers.